lib/linguist/languages.yml

```diff
@@ -449,6 +449,14 @@ Cap'n Proto:
   - .capnp
   ace_mode: text
 
+CartoCSS:
+  aliases:
+  - Carto
+  extensions:
+  - .mss
+  ace_mode: text
```

Comment:
Oh cool, we could use that. Can you take a look at the section on [adding a new grammar](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#syntax-highlighting-looks-wrong) and open a new pull request? We'll also need https://github.com/yohanboniface/carto-atom to add a license to their repo before we can add it.